packages/client/bin/cli.ts

```diff
@@ -184,12 +188,31 @@ async function run() {
     chain = args.network
   }
 
-  // TODO: map chainParams (and lib/util.parseParams) to new Common format
-  // and pass into common constructor below
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const chainParams = args.params ? await parseParams(args.params) : args.network
+  // configure common based on args given
+  let common: Common = {} as Common
+  // Use custom chain parameters file if specified
+  if (args.params) {
+    try {
+      const params = JSON.parse(fs.readFileSync(args.params, 'utf-8'))
```

Comment:
This `params` naming is obviously just some relic from the past. But we can keep for this PR, will update when I do the Common chain file format integration.

Some other note: the whole naming of methods in `util/parse.ts` is not so optimal (not introduced by this PR but rather taken from the previous work). Some methods are labelled `parse*` and are doing something completely different. Geth-specific functionality is sometimes labelled accordingly, sometimes not. This should be updated at some point (especially the two exported methods `parseParams` and `parseGenesisState` should get some Geth prefix (`parseGethParams`).